prog/vm/nexus.rb

```diff
@@ -159,6 +159,14 @@ def before_run
         hop_destroy
       end
     end
+
+    when_recreate_set? do
+      if strand.label != "destroy"
```

Comment:
```suggestion
      unless strand.label == "destroy"
```

Optional, but using unless with == is more idiomatic in Ruby than if with !=.